src/DropdownVisibility.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+use Glpi\Application\View\TemplateRenderer;
+use Glpi\Dashboard\Widget;
+
+class DropdownVisibility extends CommonDBChild
+{
+    public static $mustBeAttached  = false; //no idea why.
```

Comment:
Do you remember what was the issue here?

Reply:
Absolutely not :/ I'll try to remove it and make some tests to see.

Reply:
Removing that beaks State tests (even if I log user before).

```
> There are 2 failures:

=> tests\units\State::testVisibility():
In file /var/www/html/private/glpi/tests/functional/State.php on line 105, array() failed: array(0) has size 0, expected size 2

=> tests\units\State::testIsStateVisible():
In file /var/www/html/private/glpi/tests/functional/State.php on line 165, boolean() failed: boolean(false) is not true
-Expected
+Actual
@@ -1 +1 @@
-bool(true)
+bool(false)
```